pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "cgen"
-version = "2020.2"
+version = "2024.0"
```

Comment:
.1?

Reply:
I think I added '2024.0' to other projects too :confounded:

Reply:
Eh, not a huge deal. :) I'll get to it when (if?) things actually gets released.